types/node/index.d.ts

```diff
@@ -2354,7 +2354,7 @@ declare module "net" {
         unref(): void;
         ref(): void;
 
-        remoteAddress: string;
```

Comment:
Is it only the remote address that is cleared on (e.g.) disconnection?  Not the remote family or port?  The docs don't indicate that either of those can be null, but they seem like they'd be managed as a group.